spec/models/piece_spec.rb

```diff
@@ -1,6 +1,33 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
+  describe 'move_to! method' do
+    let(:game) { FactoryGirl.create(:game) }
+    it 'should return error if target piece has same color as current piece' do
+      current_piece = FactoryGirl.create(:piece, color: 0, game_id: game.id)
+      target_piece = FactoryGirl.create(:piece, x_coordinate: 4, y_coordinate: 4, color: 0, game_id: game.id)
```

Comment:
since the variable `target_piece` isn't called in this test. You don't need it. This line can be...

```Ruby
FactoryGirl.create(:piece, x_coordinate: 4, y_coordinate: 4, color: 0, game_id: game.id)
```

instead